server/models/Request.js

```diff
@@ -13,29 +13,29 @@ const requestSchema = new mongoose.Schema({
   },
   start: {
     type: Date,
-    required:true
+    required: true
```

Comment:
@B1umenkrAn2 This file is conflicting with the main branch now that you've merged in the model. Please address the conflict